### PR TITLE
{vis}[intel/2018a] p4vasp v0.3.30 + deps w/ Python 2.7.14

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2632.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3be21aac800edad75877e3e79a62e3b0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2054.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/be8656818d378f145cd6a7e660fd949e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node3170.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8866e64c7343bafb42fdcb6a29129584 for a full test report.

**Comment by boegel** [comment]
(close/open to retrigger Travis)

**Comment by boegel** [comment]
(re-triggering Travis checks again...)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2695.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0bfcd2ba9aceb6f57866ce230028e6c5 for a full test report.

File: easybuild/easyconfigs/p/PyGObject/PyGObject-2.28.7-intel-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'PyGObject'
+version = '2.28.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pygtk.org/'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/PyGObject/PyGObject-2.28.7-intel-2018a-Python-2.7.14.eb]:
```suggestion
homepage = 'https://pygobject.readthedocs.io/'
```

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/36cdcd9a8ce022262beb216d7f7c897c for a full test report.

File: easybuild/easyconfigs/l/libxml2-python/libxml2-python-2.9.7-intel-2018a-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'EB_libxml2'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxml2-python/libxml2-python-2.9.7-intel-2018a-Python-2.7.14.eb]: Using `PythonPackage` here would be cleaner, see #7261

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node3154.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/02be1673b981d7e4d4e3cac155d8a048 for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Good to go?

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/c4b9e1c091013d524f16d9aa8a38a08b for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**